### PR TITLE
Issue 441: synced and updated componet tables

### LINKED ISSUE #441: Bug: two different component tables on two pages
### Describe the bug.

There are two (different) component tables.  We should compare/merge/delete/refresh/update versions.

- https://open-horizon.github.io/docs/getting_started/components/
- https://open-horizon.github.io/docs/getting_started/overview_oh/

### Describe the steps to reproduce the behavior

_No response_

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating environment

All

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Demopans**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #441

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by johnwalicki** [comment]
Thanks - this is going in the correct direction!  Merging the tables is an important step.

The next step would be to remove the table completely from the overview_oh.md and only keep the table in the components.md file.

@Demopans - can you push another commit into this PR, and remove the table from overview_oh.md ?

**Comment by johnwalicki** [comment]
Can you also add a table caption (required to comply with Accessibility guidelines) in the components.md file?
Here is an example:
```
{: caption="Table 1. {{site.data.keyword.ieam}} components" caption-side="top"}
```

**Comment by Demopans** [comment]
Sure. I'll see to it

File: docs/getting_started/components.md [hunk 1]
```diff
@@ -24,12 +24,20 @@ nav_bar: 1
...
-|Exchange API|2.87.4-605|The Exchange provides the REST API that holds all the definitions (patterns, policies, services, and so on) used by all the other components in {{site.data.keyword.ieam}}.|
-|Secure Device Onboard|1.11.14-40|The SDO component enables technology that is created by Intel to make it simple and secure to configure edge devices and associate them with an edge management hub.|
-|Cluster Agent|{{site.data.keyword.anax_ver}}|This is a container image, which is installed as an agent on edge clusters to enable node workload management by {{site.data.keyword.ieam}}.|
-|Device Agent|{{site.data.keyword.anax_ver}}|This component is installed on edge devices to enable node workload management by {{site.data.keyword.ieam}}.|
-|Secrets Manager|1.1.1-208|The Secrets Manager is the repository for secrets deployed to edge devices, enabling services to securely receive credentials used to authenticate to their upstream dependencies.|
+[//]: # (Pull from release tags hopefully?)
+
+{: caption="Table 1. {{site.data.keyword.ieam}} components" caption-side="top"}
```
Comment (johnwalicki) [inline docs/getting_started/components.md:29]: My bad for not being specific.  The table caption should be directly after the table. No extra blank lines.
```
| last entry | 1.0.0 | description |
{: caption="Table 1. {{site.data.keyword.ieam}} components" caption-side="top"}
```

Can you force push another commit into this PR and move the caption line? Thx

File: docs/getting_started/overview_oh.md [hunk 2]
```diff
@@ -45,21 +45,7 @@ The {{site.data.keyword.ieam}} management hub is designed specifically for edge
...
-|SDO|1.8.6|The Secure Device Onboarding (SDO) service enables SDO-enabled edge devices to be configured with zero touch.|
-|Secrets Manager|1.0.0|The Secrets Manager is the repository for secrets deployed to edge devices, enabling services to securely receive credentials used to authenticate to their upstream dependencies.|
-|**Edge node**| |Any edge device, edge cluster, or edge gateway where edge computing takes place.|
-|Edge cluster agent|2.27.0|The agent that is installed on edge clusters to enable node workload management by {{site.data.keyword.ieam}}.|
-|Edge device agent|2.27.0|The agent that is installed on edge devices to enable node workload management by {{site.data.keyword.ieam}}.|
-|ESS|1.4.1|The edge node part of MMS that makes AI models and files available to the edge services.|
-|Example edge services|2.27.0|Edge service examples that are useful when exploring {{site.data.keyword.ieam}} and learning how to write your own services.|
+{{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}) includes [several components](../getting_started/components.md) that are bundled with the product. Please see (Getting Started/Components.md).
```
Comment (johnwalicki) [inline docs/getting_started/overview_oh.md:48]: Let's not call Open Horizon a "product".  It's an open source project.
Here is a proposed rewrite of this sentence:
```
For more information about components that are bundled with {{site.data.keyword.ieam}}, see [Components](components.md).
```
Note that both the overview_oh.md and the components.md are in the same ../getting_started/ directory so we don't actually need to use a relative path.

**Review by johnwalicki — CHANGES_REQUESTED** [review]
@Demopans  See below suggestions.  Thanks for the edits!

**Review by johnwalicki — APPROVED** [review]

**Comment by johnwalicki** [comment]
Thanks !!

**Pull request merged**